Pytest fails on brand new source installation WSL2

Environment data
OS (Windows | Mac | Linux distro) and version: 5.4.72-microsoft-standard-WSL2 Description: Ubuntu 20.04.1 LTS Release: 20.04
Python and/or Anaconda version: Python 3.8.5 conda 4.9.2
Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
idaes-pse version: idaes, version 1.9.0.dev0
pyomo version: Pyomo 6.0 (devel {master}) (CPython 3.8.5 on Linux 5.4.72-microsoft-standard-WSL2)
[idaes_test.txt](https://github.com/IDAES/idaes-pse/files/6094380/idaes_test.txt)

After doing the installation from source, and calling `idaes get-extensions`, y ran the tests via pytests. Some of them failed, see the logs below:
Full test output attached.
```
=============================================== short test summary info ================================================
FAILED idaes/commands/tests/test_commands.py::test_examples_cli_default_version - assert 0 == -1
FAILED idaes/commands/tests/test_commands.py::test_examples_install_src - FileExistsError: [Errno 17] File exists: '/...
FAILED idaes/core/tests/test_control_volume_0d.py::test_add_total_element_balances_lineraly_dependent - assert 30 == 21
FAILED idaes/core/tests/test_control_volume_1d.py::test_add_total_element_balances_lineraly_dependent - assert 30 == 21
FAILED idaes/core/util/tests/test_phase_equilibria.py::test_Txy_data_no_dew - assert [191.46973758...6254729593885] =...
FAILED idaes/generic_models/unit_models/distillation/tests/test_tray_column.py::TestBTXIdeal::test_solution - assert ...
================================ 6 failed, 3749 passed, 3 skipped in 911.96s (0:15:11) =================================
```
Each detailed test output is:
```
E       assert 0 == -1
E        +  where 0 = <Result okay>.exit_code

idaes/commands/tests/test_commands.py:131: AssertionError
```
```
>       src_dir.mkdir()

idaes/commands/tests/test_commands.py:259:
```
```
E           FileExistsError: [Errno 17] File exists: '/home/bernalde/.idaes/_scratch/examples_install_src/src'

../../../anaconda3/envs/optim/lib/python3.8/pathlib.py:1284: FileExistsError
```
```
E           assert 30 == 21
E            +  where 30 = <LogRecord: pyomo.core, 30, /home/bernalde/repos/pyomo/pyomo/common/deprecation.py, 72, "DEPRECATED: The iteritems met...ated. Use dict.items().\n(deprecated in TBD)\n(called from /home/bernalde/repos/pyomo/pyomo/core/base/component.py:838)">.levelno
E            +  and   21 = idaeslog.INFO_LOW

/home/bernalde/repos/IDAES/idaes-pse/idaes/core/tests/test_control_volume_0d.py:1846: AssertionError
```
```
E           assert 30 == 21
E            +  where 30 = <LogRecord: pyomo.core, 30, /home/bernalde/repos/pyomo/pyomo/common/deprecation.py, 72, "DEPRECATED: The iteritems met...ated. Use dict.items().\n(deprecated in TBD)\n(called from /home/bernalde/repos/pyomo/pyomo/core/base/component.py:838)">.levelno
E            +  and   21 = idaeslog.INFO_LOW

/home/bernalde/repos/IDAES/idaes-pse/idaes/core/tests/test_control_volume_1d.py:2588: AssertionError
```
```
E       assert [191.46973758...6254729593885] == [185.6468 ± 1...625 ± 1.0e-04]
E         At index 0 diff: 191.46973758055483 != 185.6468 ± 1.0e-04
E         Right contains one more item: 331.1625 ± 1.0e-04
E         Use -v to get the full diff

/home/bernalde/repos/IDAES/idaes-pse/idaes/core/util/tests/test_phase_equilibria.py:208: AssertionError
```
```
E       assert 377.28 ± 1.0e-02 == 377.2929829205527
E        +  where 377.28 ± 1.0e-02 = <function approx at 0x7f694a7c6040>(377.28, abs=0.01)
E        +    where <function approx at 0x7f694a7c6040> = pytest.approx
E        +  and   377.2929829205527 = value(<pyomo.core.base.var.SimpleVar object at 0x7f6929a2d580>)

/home/bernalde/repos/IDAES/idaes-pse/idaes/generic_models/unit_models/distillation/tests/test_tray_column.py:247: AssertionError
```

--- a/idaes/commands/examples.py
+++ b/idaes/commands/examples.py
@@ -399,7 +399,7 @@ def download_contents(target_dir, version):
     # move the REPO_DIR subdirectory into the target dir
     subdir = Path(tempdir.name) / f"{REPO_NAME}-{version}" / REPO_DIR
     _log.debug(f"move {subdir} -> {target_dir}")
-    os.rename(str(subdir), str(target_dir))
+    shutil.move(str(subdir), str(target_dir))
     zipf.close()
 
 
@@ -565,7 +565,7 @@ def install_src(version, target_dir):
         raise InstallError(f"error writing temporary __init__.py files: {err}")
     # temporarily rename target directory to the package name
     _log.info(f"rename {target_dir} -> {examples_dir}")
-    os.rename(target_dir, examples_dir)
+    shutil.move(target_dir, examples_dir)
     # if there is a 'build' directory, move it aside
     build_dir = root_dir / 'build'
     if build_dir.exists():
@@ -573,7 +573,7 @@ def install_src(version, target_dir):
         random_letters = str(uuid1())
         moved_build_dir = f"{build_dir}.{random_letters}"
         _log.debug(f"move existing build dir to {moved_build_dir}")
-        os.rename(str(build_dir), moved_build_dir)
+        shutil.move(str(build_dir), moved_build_dir)
     else:
         _log.debug("no existing build directory (nothing to do)")
         moved_build_dir = None
@@ -603,7 +603,7 @@ def install_src(version, target_dir):
             _log.debug(f"(setup) {line}")
     # name the target directory back to original
     _log.info(f"rename '{examples_dir}' to  '{target_dir}'")
-    os.rename(examples_dir, target_dir)
+    shutil.move(examples_dir, target_dir)
     # remove the empty __init__.py files
     _log.info("remove temporary __init__.py files")
     for d in pydirs:
@@ -618,7 +618,7 @@ def install_src(version, target_dir):
         _log.warning(f"failed to remove build directory {build_dir}: {err}")
     if moved_build_dir is not None:
         _log.info(f"restore build dir '{build_dir}' from '{moved_build_dir}'")
-        os.rename(moved_build_dir, build_dir)
+        shutil.move(moved_build_dir, build_dir)
     # restore previous args
     sys.argv = saved_args
     # change back to previous directory

--- a/idaes/commands/tests/test_commands.py
+++ b/idaes/commands/tests/test_commands.py
@@ -118,7 +118,7 @@ def test_examples_cli_download(runner, tempdir):
 
 
 @pytest.mark.integration()
-def test_examples_cli_default_version(runner, tempdir):
+def test_examples_cli_explicit_version(runner, tempdir):
     dirname = str(tempdir / "examples")
     result = runner.invoke(examples.get_examples, ["-d", dirname, "-I", "-V", "1.5.0"])
     assert result.exit_code == 0
@@ -128,7 +128,7 @@ def test_examples_cli_default_version(runner, tempdir):
 def test_examples_cli_default_version(runner, tempdir):
     dirname = str(tempdir / "examples")
     result = runner.invoke(examples.get_examples, ["-d", dirname])
-    assert result.exit_code == -1
+    assert result.exit_code == 0
 
 
 @pytest.mark.integration()
@@ -149,7 +149,8 @@ def test_examples_cli_copy(runner, tempdir):
     assert result.exit_code == -1
     # local dir exists, no REPO_DIR in it
     src_dir = tempdir / "examples-dev"
-    src_dir.mkdir()
+    if not src_dir.exists():
+        src_dir.mkdir()
     result = runner.invoke(
         examples.get_examples, ["-d", dirname, "--local", str(src_dir), "-I"]
     )
@@ -256,12 +257,15 @@ def test_examples_install_src():
     _log.debug(f"install_src: curdir={os.curdir}")
     # create fake package
     src_dir = tempdir / "src"
-    src_dir.mkdir()
+    if not src_dir.exists():
+        src_dir.mkdir()
     m1_dir = src_dir / "module1"
-    m1_dir.mkdir()
+    if not m1_dir.exists():
+        m1_dir.mkdir()
     (m1_dir / "groot.py").open("w").write("print('I am groot')\n")
     m2_dir = m1_dir / "module1_1"
-    m2_dir.mkdir()
+    if not m2_dir.exists():
+        m2_dir.mkdir()
     (m2_dir / "groot.py").open("w").write("print('I am groot')\n")
     # install it
     examples.install_src("0.0.0", src_dir)

--- a/idaes/commands/tests/test_commands.py
+++ b/idaes/commands/tests/test_commands.py
@@ -149,8 +149,7 @@ def test_examples_cli_copy(runner, tempdir):
     assert result.exit_code == -1
     # local dir exists, no REPO_DIR in it
     src_dir = tempdir / "examples-dev"
-    if not src_dir.exists():
-        src_dir.mkdir()
+    src_dir.mkdir(exist_ok=True)
     result = runner.invoke(
         examples.get_examples, ["-d", dirname, "--local", str(src_dir), "-I"]
     )
@@ -257,15 +256,12 @@ def test_examples_install_src():
     _log.debug(f"install_src: curdir={os.curdir}")
     # create fake package
     src_dir = tempdir / "src"
-    if not src_dir.exists():
-        src_dir.mkdir()
+    src_dir.mkdir(exist_ok=True)
     m1_dir = src_dir / "module1"
-    if not m1_dir.exists():
-        m1_dir.mkdir()
+    m1_dir.mkdir(exist_ok=True)
     (m1_dir / "groot.py").open("w").write("print('I am groot')\n")
     m2_dir = m1_dir / "module1_1"
-    if not m2_dir.exists():
-        m2_dir.mkdir()
+    m2_dir.mkdir(exist_ok=True)
     (m2_dir / "groot.py").open("w").write("print('I am groot')\n")
     # install it
     examples.install_src("0.0.0", src_dir)

